src/Console/Shell.php

```diff
@@ -242,12 +249,14 @@ public function startup()
      */
     protected function _welcome()
     {
+        $phpInfo = explode('~', phpversion());
+        $this->phpVersion = (isset($phpInfo[0]) ? $phpInfo[0] : phpversion());
```

Comment:
The isset is not necessary as it will always have a 0 element after an explode() of any content input.